cocos/ui/sub-context-view.ts

```diff
@@ -106,25 +106,20 @@ export class SubContextView extends Component {
 
     @serializable
     private _fps = 60;
-    private _sprite: Sprite | null;
-    private _imageAsset: ImageAsset;
-    private _texture: Texture2D;
+    private _sprite: Sprite | null = null;
+    private _imageAsset: ImageAsset = new ImageAsset();
+    private _texture: Texture2D = new Texture2D();
     private _updatedTime = 0;
     private _updateInterval = 0;
-    private _openDataContext: any;
-    private _content: Node;
+    private _openDataContext: any = null;
+    private _content: Node = new Node('content');
```

Comment:
Why not use declare?

Reply:
Only use `declare` when properties are initialized in `constructor`. Otherwise, it doesn't need to use `declare` keyword.

Reply:
They are initialized in constructor.